crates/walrus-service/bin/client.rs

```diff
@@ -17,6 +18,18 @@ use walrus_sui::client::retry_client::RetriableRpcError;
 /// The version of the Walrus client.
 pub const VERSION: &str = walrus_service::utils::version!();
 
+/// The build time of the Walrus client.
+pub const BUILD_TIME: DateTime<Utc> = {
+    let Ok(timestamp) = i64::from_str_radix(env!("BUILD_TIME"), 10) else {
+        panic!("BUILD_TIME should be set to a valid UNIX timestamp by the build script");
+    };
+    if let Some(build_time) = DateTime::from_timestamp(timestamp, 0) {
+        build_time
+    } else {
+        panic!("BUILD_TIME should be set to a valid UNIX timestamp by the build script");
+    }
+};
```

Comment:
nit
```suggestion
    let timestamp = i64::from_str_radix(env!("BUILD_TIME"), 10).expect("BUILD_TIME should be set to a valid UNIX timestamp by the build script");
    DateTime::from_timestamp(timestamp, 0).expect("BUILD_TIME should be set to a valid UNIX timestamp by the build script")
};
```

Reply:
I see that `expect` is non-const, so this won't work. Resolving this.

Reply:
> I see that `expect` is non-const, so this won't work. Resolving this.

Yes, that was the reason for the more verbose version.